account-kit/react/src/components/auth/card/error/connection-error.tsx

```diff
@@ -1,106 +1,46 @@
-import { useEffect, useMemo } from "react";
+import { useEffect } from "react";
 import { ls } from "../../../../strings.js";
 import { Button } from "../../../button.js";
-import { ConnectionFailed as PasskeyConnectionFailed } from "../../../../icons/connectionFailed.js";
-import { Timeout } from "../../../../icons/timeout.js";
-import { EOAWallets, type ConnectionErrorProps } from "./types.js";
-import { WalletIcon } from "./icons/wallet-icon.js";
-import { OAuthConnectionFailed } from "../../../../icons/oauth.js";
-import { capitalize } from "../../../../utils.js";
+import type { ConnectionErrorProps } from "./types.js";
 import { disconnect } from "@account-kit/core";
 import { useAlchemyAccountContext } from "../../../../context.js";
 
-export const walletTypeConfig = [
-  { name: "Coinbase Wallet", key: EOAWallets.COINBASE_WALLET },
-  { name: "MetaMask", key: EOAWallets.METAMASK },
-  { name: "Wallet Connect", key: EOAWallets.WALLET_CONNECT },
-];
-
 export const ConnectionError = ({
-  connectionType,
-  oauthProvider,
-  EOAConnector,
+  headerText,
+  bodyText,
+  tryAgainCTA,
```

Comment:
we should have defaults for these since the following seems pretty common
```
        headerText={errorMessage.heading}
        bodyText={errorMessage.body}
        tryAgainCTA={errorMessage.tryAgainCTA}
```

Reply:
oh wait nm, no I see the usage now. This is fine